TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -82,6 +82,17 @@ import UIKit
             return self._brokerCenterService!
         }
     }
+    
+    internal static var _streamingMarketDataService: StreamingMarketDataService?
+    public static var streamingMarketDataService: StreamingMarketDataService? {
+        get {
+            return self._streamingMarketDataService
```

Comment:
If there is no pre-condition check needed then you don't need the getter/setter.